components/kserve/kserve.go

```diff
@@ -0,0 +1,42 @@
+package kserve
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/opendatahub-io/opendatahub-operator/components"
+	"github.com/opendatahub-io/opendatahub-operator/pkg/common"
+	"github.com/opendatahub-io/opendatahub-operator/pkg/deploy"
+)
+
+const (
+	ComponentName = "kserve"
+	Path          = deploy.DefaultManifestPath + "/" + ComponentName + "/base"
```

Comment:
are we going to have a https://github.com/opendatahub-io/odh-manifests/kserve folder? or these config will live under` odh-manifests/model-mesh` ?

Reply:
We are going to have https://github.com/opendatahub-io/odh-manifests/kserve